sdk/servicebus/azure-servicebus/azure/servicebus/exceptions.py

```diff
@@ -193,11 +244,11 @@ class OperationTimeoutError(ServiceBusError):
 
 
 class MessageError(ServiceBusError):
-    """A message failed to send because the message is in a wrong state"""
+    """An error occurred when operation on message failed because the message is in a wrong state."""
```

Comment:
```suggestion
    """An error occurred when an operation on a message failed because the message is in an incorrect state."""
```

(grammar nit re: first half of the clause)